graphql/schemabuilder/pagination.go

```diff
@@ -506,30 +510,30 @@ func (sb *schemaBuilder) buildPaginatedField(typ reflect.Type, m *method) (*grap
 				if !ok {
 					return nil, fmt.Errorf("arguments should implement ConnectionArgs")
 				}
-				funcCtx.hasArgs = val.Args != nil
-				if funcCtx.hasArgs {
+				c.hasArgs = val.Args != nil
+				if c.hasArgs {
 					argsVal = reflect.ValueOf(val.Args).Elem().Interface()
 				}
 			}
 
-			in := funcCtx.prepareResolveArgs(source, argsVal, ctx)
+			in := c.prepareResolveArgs(source, argsVal, ctx)
 
 			// Call the function.
 			out := fun.Call(in)
 
-			return funcCtx.extractPaginatedRetAndErr(nodeKey, out, args, retType, embedsArgs, returnsPageInfo)
+			return c.extractPaginatedRetAndErr(nodeKey, out, args, retType, embedsArgs, returnsPageInfo)
 
 		},
 		Args:           args,
 		Type:           retType,
 		ParseArguments: argParser.Parse,
-		Expensive:      funcCtx.hasContext,
+		Expensive:      c.hasContext,
 	}
 
 	return ret, nil
 }
 
-func (funcCtx *funcContext) extractPaginatedRetAndErr(nodeKey string, out []reflect.Value, args interface{}, retType graphql.Type, embedsArgs bool, returnsPageInfo bool) (interface{}, error) {
+func (c *connectionContext) extractPaginatedRetAndErr(nodeKey string, out []reflect.Value, args interface{}, retType graphql.Type, embedsArgs bool, returnsPageInfo bool) (interface{}, error) {
```

Comment:
`extractPaginatedReturnAndErr`?

Reply:
I'll probably remove the Paginated part of the name since we're touching this